src/app/components/ats/candidates/new-candidate/new-candidate.component.html

```diff
@@ -37,43 +37,43 @@ <h2 *ngIf="isValidProfileImage" class="center-bold">Profile Picture</h2>
                         <mat-label>First name</mat-label>
                         <input matInput type="text" formControlName="name" />
                         <mat-error *ngIf="newCandidateForm.controls['name'].hasError('required')">
-                            First name is required
+                            First Name is Required
                         </mat-error>
                     </mat-form-field>
                     <mat-form-field appearance="outline" class="col-md-12 col-lg-7 my-2">
                         <mat-label>Last name</mat-label>
                         <input matInput type="text" formControlName="surname" />
                         <mat-error *ngIf="newCandidateForm.controls['surname'].hasError('required')">
-                            Last name is required
+                            Last Name is Required
                         </mat-error>
                     </mat-form-field>
                     <mat-form-field appearance="outline" class="col-md-12 col-lg-7 my-1">
                         <mat-label>Email</mat-label>
                         <input matInput type="email" formControlName="email" (change)="onEmailChange($event)" />
                         <mat-error *ngIf="newCandidateForm.controls['email'].hasError('pattern')">
-                            Please enter a valid email address
+                            Please Enter a Valid Email Address
                         </mat-error>
                         <mat-error *ngIf="newCandidateForm.controls['email'].hasError('required')">
-                            Email address is required
+                            Email Address is Required
                         </mat-error>
                     </mat-form-field>
                     <div class="col-md-12 col-lg-7 my-1">
                         <h3 *ngIf="candidateExists" class="warning-message">Duplicate Canidate found</h3>
                         <h3 *ngIf="isBlacklisted" class="warning-message">Blacklisted Canidate</h3>
-                        <h3 *ngIf="candidateWarning" class="warning-message">This candidate has a warning</h3>
-                        <p *ngIf="isBlacklisted">This candidate has been blacklisted from the company.<br>Go to their profile to see the reason or lift the blacklisting</p>
-                        <p *ngIf="candidateExists">The candidate already exists in our system</p>
-                        <p *ngIf="candidateWarning">This candidate has recieved a warning in the system</p>
+                        <h3 *ngIf="candidateWarning" class="warning-message">This Candidate has a Warning</h3>
+                        <p *ngIf="isBlacklisted">This Candidate has been Blacklisted From the Company.<br>Go to their Profile to See the Reason or Lift the Blacklisting</p>
```

Comment:
Just check and see if pronouns and words like "is" or "has" should also have the first letter be capitalized.